data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,848 @@
+			`	Darius takes a while to respond. "Leon. It's been a while."`
+			`	Leon snorts. "That's to say the least. Is Sora with you, or did you crawl here by yourself?"`
+			choice
+				`	"She's helping with some negotiations."`
+					goto you
+				`	(Let Darius do the talking.)`
+			`	"She's with someone from the Consonance, negotiating with the miners here," says Darius.`
+			label you
+			`	"And who's this?" Leon says, looking at you. "How'd you end up here, of all places?"`
+			choice
+				`	"Captain <last>. I was stranded here, just like you."`
+				`	"<first>. Pleased to meet you."`
+			`	You quickly explain how you found yourself in Avgi space, and Leon nods. "That explains the questions the good Captain was asking," he says, nodding at Darius. "I've spent years collecting this information, from hearing the official narrative to dredging the corners of Avgi space for rumors."`
+			`	He leans in. "So, before I tell you this, one condition. I've spent months looking for a way to get a ship to take me to... well, you'll see. But not a lot of captains are willing to take their ships out of the Tangled Shroud and through the Aberrant blockade, so if this leads anywhere, I want a bunk on your ship."`
+			choice
+				`	"Agreed."`
+					goto agreed
+				`	"No deal."`
+			`	You open your mouth to reply, but Darius beats you to the punch. "Of course, Leon."`
+
+			label agreed
+			`	"Good. Well, it more or less comes down to this: the Incursion didn't come out of nowhere. Everyone says they came from beyond the furthest reaches of Avgi exploration, but that's not quite true. The first to encounter them was a Polari scout ship, which went down within minutes against their weapons. But not before they sent out a single transmission, garbled and not making a lot of sense."`
+			`	He pauses, taking a drink of what looks like water. "They said they had found a gateway of sorts, an ancient alien artifact built by an unknown Predecessor species. Somehow, they had activated it, and that's where the first Aberrant came from."`
+			choice
+				`	"And you want to use this gateway as a way out?"`
+				`	"Where do you think this gateway leads?"`
+			`	"I don't know where it leads, but it'll be somewhere other than Avgi space. Maybe somewhere your Jump Drive can work, which would let us eventually make our way back to human space. But it's better than sitting here and twiddling our thumbs for another six years."`
+			`	"Where is this gateway?" Darius asks.`
+			`	"The Aberrant came from the East, beyond the furthest reaches of Avgi space. A place the Avgi always had trouble exploring, due to strange energy storms periodically ionizing entire systems. But it's apparently recognizable by the red nebula suffusing it, just as the Twilight is surrounded by a cluster of blue nebulas."`
+			`	"There's no point in waiting, then. If there's a chance, even a small one, we should check it out," says Darius, looking hopeful for the first time since you've met him.`
```

Comment:
Maybe there should be some option here to tell them about the Ember Waste if you’ve found it?